app/views/users/posts/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="posts">
+  <div class="userposts">
+    Showing <%= @user.email %>'s recent posts
```

Comment:
throw this text in a `<p>` tag at minimum. i am not sure the hierarchy of the headings on the page this will end up, but i would recommend putting it in an `<h2>` or another heading instead of the paragraph `<p>` tag

Reply:
A <p> created an unwanted gap around the line of text. Is text directly in a span also bad practice? If so, I'll put it in a <p> and mess with the styling to get rid of the automatic margins.
![Screen Shot 2019-06-07 at 2 10 53 PM](https://user-images.githubusercontent.com/37525354/59206189-6e9fa680-8b72-11e9-99ad-3064bdd9ca67.png)

Reply:
yes, any text inside an element that isnt reserved for text isnt good practice. usually browsers add margins to text elements whether you want them or not, so something worthwhile may be adding a [browser reset stylesheet](https://meyerweb.com/eric/tools/css/reset/) to your project to not have things style in ways you dont control yourself. that example is just one of many out there